orbstation/code/customisation/hair.dm

```diff
@@ -128,3 +128,7 @@
 	name = "tulip"
 	icon_state = "tulip"
 	icon = 'orbstation/icons/mob/species/podperson/podperson_hair.dmi'
+
+/datum/sprite_accessory/hair/skinhead
+	name = "S.H.A.R.P."
+	icon_state = "hair_skinhead"
```

Comment:
This line isn't necessary, because it hasn't changed from the original.

Reply:
Line 134, I meant. I thought I'd only selected that one.

Reply:
yeah sure one sec